recipes/h2mm-c/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "h2mm-c" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "949ab32a3255d286fbb09b2bb9918bac21d4fd6c0cb4216dfc264bf82fa827ce" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/harripd/H2MMpythonlib/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+  host:
+    - python >=3.7
```

Comment:
```suggestion
    - python
```

Reply:
Before I commit the change, I know that this package won't work on python 3.6 because of numpy compatibility- will the other changes you made still enforce this requirement/prevent ci from failing because it won't work on Python < 3.7?

I also have inherited another conda-forge package, phconvert, and on that one, which is a much simpler package, having version specifiers didn't create the same error. So I'm confused and would like an explanation so I don't have to keep asking for help in the future.

Reply:
phconvert:

* No cython or C code. So it is `noarch: python`


Conda-forge no longer builds for python versions below 3.8. Only 3.8 through 3.11 and 3.12 after a "migration occurs".

You could add:
```
skip: true   # [py<37]
```
in the build section out of an abundance of caution, but this kind of "no-op" configuration is typically not left in place at conda-forge.

So for you, the package would build for 3.8, 3.9, 3.10, 3.11 all of which are supported.

Reply:
Great, thanks for the clarification